src/server/bin/plugin/plugin.js

```diff
@@ -0,0 +1,36 @@
+﻿var settingParser = require('./settingParser');
+var installer = require('./pluginInstaller');
+var remover = require('./pluginRemover');
+var pluginLogger = require('./pluginLogger');
+
+module.exports = function (program) {
+  function processCommand(command, options) {
+    var settings;
+    try {
+      settings = settingParser.parse(command);
+    } catch (ex) {
+      //The logger has not yet been initialized.
+      console.error(ex.message);
```

Comment:
It should already be respecting the silent option. The stdout and stserr streams are passed to the logger. The logger then checks the silent option to determine whether the output should be piped. Any errors ignore the silent option.